### DISCUSSION TITLE
🐞 Cannot send to 10 digit number

### DISCUSSION THREAD

**Discussion by pekalongan** [aldinokemal/go-whatsapp-web-multidevice#154] [Q&A] [answered]
### What happened?

Cannot send to 10 digit number 

### Reproduction Steps

```bash
1. Open go-whatsapp-web-multidevice instance
2. Send message
3. 628xxxxxxxx (10 digits number)
4. Error phone 628xxxxxxxx is not on whatsapp
...
```

### Screenshots or Additional Context

![image](https://github.com/aldinokemal/go-whatsapp-web-multidevice/assets/1832716/8bdf379e-ed60-4ee6-bddf-2451d3c94ad6)

### Application Version

4.13.0

### Verification

- [X] I have checked for existing closed issues and discussions.

**Comment by aldinokemal** [comment]
There is no phone digit number validation. Instead, there is WhatsApp checking whether a phone exists.

**Comment by suharyadi2112** [comment]
https://github.com/aldinokemal/go-whatsapp-web-multidevice/discussions/229

@pekalongan

**Comment by daru** [comment]
Same here, can't send messages to 10 digits number, while I can chat normally to those number using phone or desktop app.

**Comment by aldinokemal** [comment, marked as answer]
https://github.com/aldinokemal/go-whatsapp-web-multidevice/discussions/229#discussioncomment-11897547

duplicate
